src/auditlog/mixins.py

```diff
@@ -39,26 +49,61 @@ def resource_url(self, obj):
     resource_url.allow_tags = True
     resource_url.short_description = 'Resource'
 
+    def remote_addr_url_w(self, obj):
+        if obj.remote_addr is None:
+            return None
+        link = "https://ipinfo.io/" + str(obj.remote_addr)
```

Comment:
this looks like code that is custom to your website/domain, perhaps remove this function